src/System.Xml.XDocument/tests/Streaming/StreamingOutput.cs

```diff
@@ -3,617 +3,571 @@
 
 using System;
 using System.IO;
-using System.Linq;
-using System.Reflection;
-using System.Collections;
 using System.Collections.Generic;
-using System.Text;
+using System.Linq;
 using System.Xml;
 using System.Xml.Linq;
+
 using System.Xml.XmlDiff;
-using Microsoft.Test.ModuleCore;
+using CoreXml.Test.XLinq;
 
-namespace CoreXml.Test.XLinq
+using Xunit;
+
+namespace XDocumentTests.Streaming
```

Comment:
@Maxwe11 - It's supposed to be ${namespaceOfTypeUnderTest}.Tests, so probably `System.Xml.Linq.Tests`?